### ISSUE TITLE
can't change language

### ISSUE THREAD

**Issue by mihstaz** [ValveSoftware/Proton#5868]
Steam support told me to make a ticket here after they were unable to help me.
The issue is that I can't change the language of the Borderlands games to german from an english steam client on english ubuntu .
No starting parameters, no changing settings in the client, no reinstalling has helped.
Also, some of the DLCs refuse to download, even though the rest of the game runs fine through proton experimental (or older versions).
client api v020
package versions 1653101165
ubuntu 22.04 LTS
kernel 5.15.0-33

**Comment by kisak-valve** [comment]
Hello @mihstaz, https://github.com/ValveSoftware/Proton/issues/3111 for your second snafu. Go to `Steam` -> `Settings` -> `Steam Play` and check the `Enable Steam Play for all other titles` checkbox.

**Comment by mihstaz** [comment]
@kisak-valve Hi, steam proton is already enabled for all titles (and, might I add, has mostly been a wonderful experience for me), but the issue still persists. Not all DLCs can be used and the language of the games is always english.

**Comment by alasky17** [comment]
@mihstaz Could you list specifically which DLCs are not working and which ones are? And also the specific titles that you have tried where switching from English to German fails?

**Comment by mihstaz** [comment]
@alasky17 Hi, so for example, the high res texture pack is marked as "not installed" in Borderlands 2, as is the creature slaughter dome and a few others. But Captain Scarlett and her pirates booty works.
the language issue is present in all borderlands games (1, 1 remake, 2, the pre-sequel, 3)

**Comment by kisak-valve** [comment]
Borderlands 2 has a Linux native version which takes priority over the global Steam Play setting in Steam. If you want to test using that game with Proton, you need to go to the per-game properties -> `Compatibility` -> check `Force the use of a specific Steam Play compatibility tool` and select a Proton version. Issues with the native Linux variant o the game should be reported directly to the game's dev(s) (or to Aspyr as the porting company in this specific case).

The `Borderlands 2 Ultra HD Texture Pack` DLC is not marked as compatible with the native Linux variant of the game, while `Borderlands 2: Creature Slaughterdome` is.

**Comment by mihstaz** [comment]
@kisak-valve I thought the "Enable steam play for all other titles" under 'steam -> settings -> steam play' enables proton for ALL games by default?
I'll enable it specifically for Bl2 and come back later with the results.

first edit: steam is currently downloading 12GB of data for Borderlands 2. Hopefully that's the remaining DLCs and language files.

**Comment by mihstaz** [comment]
second edit: Borderlands 2 is now working flawlessly. correct language and all dlc are available.
Borderlands 3 on the other hand is still english, regardless of specific compatibility options.

**Comment by alasky17** [comment]
@mihstaz I just tried Borderlands 3 on Windows, and it looks like the German language option in the Steam client doesn't change anything in the game itself (or at least not that I could tell) and the game is still in English.  I'm not sure why the developer made the German language option available, but from my testing, it doesn't appear to do anything.  Unless there are any other issues, I think we might be able to close this one out as a game bug due to lack of German language support by the developers?

**Comment by mihstaz** [comment]
I just found something else - I previously messed around with the -language german setting in start options, but as it turns out the correct parameter isnt "language", its "-culture".
I dont know why this works, but when I set -culture=de as starting option for borderlands 3 by right-clicking and selecting properties, the game launches properly.
seems like their language setting is broken and it wasnt a proton issue to begin with.
still, thanks everyone for helping me with the dlcs for Borderlands 2 and the time you took to look into this.
